packages/wonder-blocks-modal/components/standard-modal.js

```diff
@@ -36,35 +38,60 @@ type Props = {
      * provide a container element with 100% width.
      */
     footer: React.Node,
+
+    /**
+     * Called when the close button is clicked.
+     *
+     * If you're using `ModalLauncher`, you probably shouldn't use this prop!
+     * Instead, to listen for when the modal closes, add an `onClose` handler
+     * to the `ModalLauncher`.
+     *
+     * This defaults to a no-op via `defaultProps`. (When used in a
+     * `ModalLauncher`, we'll automatically add an extra listener here via
+     * `cloneElement`, so that the `ModalLauncher` can listen for close button
+     * clicks too.)
+     */
```

Comment:
Would it make sense to add a `@ignore` to this, to hide it from the docs, as well? See: https://react-styleguidist.js.org/docs/documenting.html#ignoring-props

If people find it via the code they'll read this comment and that'll be fine. It definitely seems like we want people to be using ModalLauncher and not some custom situation.

Reply:
Ooh interesting, okay! Right, part of the intent is to support that custom-y use case, but it's probably fine to make it a bit less discoverable. (And if the use case becomes common later, we can tweak discoverability later.) Thanks for the tip!!

Reply:
Done!